cmake/std/PullRequestLinuxDriver.sh

```diff
@@ -12,10 +30,29 @@ echo -e "REPO_ROOT : ${REPO_ROOT}"
 # This is the old Linux Driver (deprecated)
 #${SCRIPTPATH}/PullRequestLinuxDriver-old.sh
 
+# Both scripts will need access through the sandia
+# proxies so set them here.
+export https_proxy=http://wwwproxy.sandia.gov:80
+export http_proxy=http://wwwproxy.sandia.gov:80
+no_proxy='localhost,localnets,.sandia.gov,127.0.0.1,169.254.0.0/16,forge.sandia.gov'
```

Comment:
Why are https_proxy and http_proxy exported while no_proxy is not?

Reply:
Because I missed it. All three need to be exported.